protos/events.proto

```diff
@@ -55,6 +56,14 @@ message Register {
     repeated Interest events = 1;
 }
 
+//Rejection is sent by consumers for erroneous transaction rejection events
+//string type - "rejection"
+message Rejection {
+    string Uuid = 1;
```

Comment:
Just a nitpick, but I'd encourage changing this to just `id` as it's clear from PR #1721 that this will not always be a UUID in the future